DEVELOPMENT.md

```diff
@@ -37,6 +37,201 @@ development.
 1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
    managing development environments.
 
+### Kubernetes cluster
```

Comment:
This is not what I expected given the PR title :)

Reply:
I was expecting this to cross-link to `knative/docs` not inline it.

Reply:
it does tho - what I inlined were the pieces of the docs that were in this repo which are _not_ in knative/docs

Reply:
`/me wants better diff tools` my bad